### ISSUE TITLE
In the latest Google chrome 85.0, when arrowing right and or left, in the addres bar, NVDA speaks the first letter of the address bar control

### ISSUE THREAD

**Issue by zstanecic** [nvaccess/nvda#11529]
### Steps to reproduce:
1. focus on the blank addres bar control,
2. press right and or left arrow
### Actual behavior:
On the braille display, the caret is located at the beginning of the addres bar, instead of being at the text insertion point, and the addres bar content, concretely the first letter "a", is reported when arrowing
### Expected behavior:
NVDA should report blank
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
NVDA version alpha-20775,9a4074bc

#### Windows version:
Latest windows insider version on the beta channel
#### Name and version of other software in use when reproducing the issue:
Google chrome 85.0.4183.83 (Официальная сборка), (64 бит) russian language version
#### Other information about your system:
System is also in the russian language
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Not applicable
#### If addons are disabled, is your problem still occuring?
yes, it is still occuring
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
not applicable

**Comment by Adriani90** [comment]
Hmm I cannot reproduce it in NVDA Version: alpha-20775,9a4074bc
And Chrome 85 on a german system.
Windows 10 2004 update

Did you try without braille display? Is it still occuring?

**Comment by Adriani90** [comment]
We didn't get any updates from the initial author since years, I am closing this issue as abandoned. Please comment on it if youa re still having it and we can reopen.
